test/primitives/src/lib.rs

```diff
@@ -0,0 +1,59 @@
+// Copyright 2020 Parity Technologies (UK) Ltd.
```

Comment:
WHy do we need this?

Reply:
We need Block, Hash, AccountId, Signature of the test runtime... Maybe I can move it to the test-runtime crate directly?

Reply:
done in 78338b3